server/src/schema/statistic.ts

```diff
@@ -0,0 +1,28 @@
+import { list } from "@keystone-6/core";
+import { allowAll } from "@keystone-6/core/access";
+import { text, integer } from "@keystone-6/core/fields";
+
+import { allOperations, everyPredicate } from "../auth/access";
+
+
+export const CachedStatistic = list({
+  access: {
+    operation: {
+      ...allOperations(everyPredicate()),
+      create: allowAll,
```

Comment:
Should this be the following?
```suggestion
      ...allOperations(isAuthenticated),
      read: () => true,
```